### ISSUE TITLE
project title render inconsistencies

### ISSUE THREAD

**Issue by haraldschilly** [sagemathinc/cocalc#6334]
- In the projects listing, where all of them are, the project title is rendered as Markdown.
- The title of the project settings box for the title shows it as plain text
- The project tab selector title is also plain text

I think it should be markdown everywhere, but I wonder how confusing it is to render links...

![Screenshot from 2023-01-11 18-12-33](https://user-images.githubusercontent.com/207405/211872223-93c57d6e-358a-4938-93e2-4201979f38b7.png)

**Comment by williamstein** [comment]
I agree -- let's just make it markdown everywhere... though some thought about security and XSS attacks maybe be relevant (we do have an XSS safe mode to render Markdown now).
